client/settings/transactions-and-deposits/index.js

```diff
@@ -68,11 +70,23 @@ const TransactionsAndDeposits = () => {
 						'Issue an authorization on checkout, and capture later',
 						'woocommerce-payments'
 					) }
-					help={ __(
-						'Charge must be captured on the order details screen within 7 days of authorization, ' +
-							'otherwise the authorization and order will be canceled.',
-						'woocommerce-payments'
-					) }
+					help={
+						<span>
+							{ __(
+								'Charge must be captured on the order details screen within 7 days of authorization, ' +
+									'otherwise the authorization and order will be canceled.',
+								'woocommerce-payments'
+							) }
+							<br />
+							{ isCardPresentEligible
+								? __(
+										' Note: This setting is ignored by in-person payments. ' +
```

Comment:
Suggesting wording: `The setting is not applied to In-Person Payments (please note that In-Person Payments should be captured within 2 days of authorization).`

Reply:
I'm confused. If the setting is not applied to in-person payments (I thought it was), why would the merchant ever need to capture an in-person payment manually? Isn't this setting applicable to online AND in-person payments?

Reply:
@allendav, at this point this setting isn't applicable for IPP. The charge is captured at the moment the client pays.

Reply:
Ah. Perfect then :)